README.md

```diff
@@ -54,43 +78,49 @@ docker-compose up
 
 It's going to take 1-2 minutes until everything is initialized, this includes the postgres initialization, the caches
 announcing themselves to the bootstrap service, the publisher requesting a list of all caches from the bootstrap service
-and finally the publisher and the caches negotiating an escrow, and a distributed tracing backend store. After the escrow
-is setup you can download from the caches using `cachecash-curl` or [typescript-cachecash]:
```

Comment:
Why was this content cut?  Can't you still do these steps to test?

Reply:
Yes. Sorry this edit was done before yesterday when I made cachecash-curl work. I will change this.